tests/test_client_functional.py

```diff
@@ -1843,7 +1843,7 @@ async def handler(request):
 
     resp = await client.get("/")
     data = await resp.read()
-    assert data == b"ans"
```

Comment:
Just as an observation, I find this one a little curious. Surely 'ans' is more likely to be a misspelling for 'and' (s/d are next to each other on a qwerty keyboard...).

Reply:
`ans` is often used, especially in Matlab world since that is where it prints out as abbreviation for "answer". If you want, I can just add `ans` to be ignored by codespell, but I thought it is better to just tune up this test.

Reply:
Doesn't matter, I was just assuming you used the `-w` option and codespell has decided that 'ans' must be a misspelling of 'an', which doesn't seem like the most likely thing to me.